Gopkg.toml

```diff
@@ -72,3 +72,7 @@
 [[constraint]]
   branch = "master"
   name = "golang.org/x/sys"
+
+[[constraint]]
```

Comment:
Hm, did you add this yourself? The way I understood `dep` works is that it should figure out a dependency by inspecting the code, so this entry shouldn't be needed.

Reply:
Yeah, this is unnecessary. Still getting used to `dep` -- I forget that the Gopkg.lock file is checked in and will contain this revision anyway.